x-pack/plugins/security_solution/public/resolver/models/process_event.ts

```diff
@@ -24,6 +24,10 @@ function isValue(field: string | string[], value: string) {
   }
 }
 
+export function isTerminatedProcess(passedEvent: ResolverEvent) {
+  return eventType(passedEvent) === 'processTerminated';
```

Comment:
:grey_question: Does this just work now because of updates to the API? And it didn't before?

Reply:
`eventType(passedEvent)` had always worked, it's just that the selectors used for building the graph only looked for process creation or already running events, as we could be guaranteed that one and only one of those events would exist per process, which made graphing simpler. That's also why the code in the process event dot component did not quite work as intended.